src/actions/query.js

```diff
@@ -356,14 +365,41 @@ function msearch(
 
 const isPropertyDefined = property => property !== undefined && property !== null;
 
-function appbaseSearch(
+function getQuerySuggestionsId(componentId) {
+	return `${componentId}__suggestions`;
+}
+
+function appbaseSearch({
 	query,
 	orderOfQueries,
 	appendToHits = false,
```

Comment:
@ShahAnuj2610 Do you think that default values will work in that way?
If yes then share an example.
It seems wrong to me.

Reply:
@bietkul didn't get you. Can you share example for which it would break?